desk/lib/bolt.hoon

```diff
@@ -162,72 +160,56 @@
     =^  cards  yosh  (on-fail:ag term tang)
     [cards this]
   --
+  ::
   |%
-  ++  permissions
-    |_  [=bowl:gall state=state-0]
-    +*  parent  (sein:title our.bowl now.bowl src.bowl)
+  ++  is-allowed
+    |=  =bowl:gall
```

Comment:
At the only place where `is-allowed` was previously called, its argument was `src.bowl`. And the `bowl` had already been passed to its parent core `permissions`. So I suggest that we get rid of `permissions` and pass the `bowl` directly to `is-allowed` and let it extract `src.bowl` manually.